tcnj-acm/halo
# Sendgrid adding users to mailing list

closes #89 

Every time someone is added to the system via registration or adding an organizer, they are added to the mailing list of the campaign.

To confirm this, you'll have to be logged in to SendGrid to see the change.  

**Instructions**

1. Have the `sendgrid.env` in the same directory as manage.py 
2. Run the server and complete registration. 
3. Hit me up, I'll see if the email got added to the list. 

**Test it when adding an organizer**

1. log in to a superuser 
2. add an organizer
3. hit me up and I'll check if the organizer is added to the list
